The-Chonky-Panda/Atelier
# Finished setting up Redux

## Description
Installed Redux as our state management library. Created actions and reducers directories, store.js, Provider wrapped around our App component to provide Components access to state

## How to test
Change the initial state in store.js to a different string and check that it renders on the web page at localhost:3000

## Notes
I created some placeholder files in the actions and reducers folders to test functionality.

## Issue tracking
https://trello.com/c/Avj10Jij/15-set-up-redux
